### PR TITLE
NXP-29192: Better SIG handling for docker run

### PR DESCRIPTION
**Pull request by kevinleturc**

### CODE REVIEW

**Comment by kevinleturc** [comment]
T&P: https://qa.nuxeo.org/jenkins/job/TestAndPush/job/ondemand-testandpush-kleturc/718/

**Comment by ataillefer** [comment]
Tested
```
cd docker
mvn -nsu -f slim/pom.xml install
docker run --rm --name nuxeo  -p 8080:8080  nuxeo/slim:11.1-SNAPSHOT
```
and Ctrl + C :+1:

File: server/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeoctl [hunk 1]
```diff
@@ -233,7 +233,15 @@ while true; do
   done
   cp "$NUXEO_LAUNCHER" "$TMPLAUNCHER"
   echo [`date`] "Launcher command: $JAVA  \"-Xbootclasspath/a:$JAVA_TOOLS\" $MODULE_OPTS $LAUNCHER_DEBUG \"-Dlauncher.java.opts=$JAVA_OPTS\" \"-Dnuxeo.home=$NUXEO_HOME\" \"-Dnuxeo.conf=$NUXEO_CONF\" \"-Dnuxeo.log.dir=$LOG_DIR\" -jar \"$TMPLAUNCHER\" $@" >> "$LOG_DIR"/nuxeoctl.log
-  "$JAVA" "-Xbootclasspath/a:$JAVA_TOOLS" $MODULE_OPTS $LAUNCHER_DEBUG "-Dlauncher.java.opts=$JAVA_OPTS" "-Dnuxeo.home=$NUXEO_HOME" "-Dnuxeo.conf=$NUXEO_CONF" "-Dnuxeo.log.dir=$LOG_DIR" -jar "$TMPLAUNCHER" "$@"
+  # any signal send to a bash script is not propagated to child process by design
+  # this is an issue in Docker because nuxeoctl is PID 1 and responsible to kill its child processes
+  # thus we need to use trap/wait to forward signal to java process
```
Comment (NourNuxeo) [inline server/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeoctl:238]: then I would
```suggestion
  # thus we need to use trap/wait to forward signal to (other?) java (sub?)processes
```

Comment (kevinleturc) [reply server/nuxeo-server-tomcat/src/main/resources/tomcat/bin/nuxeoctl:238]: Here, we know we're talking about Tomcat as the java process, and there's only one.

**Pull request merged**